charts/rancher-gatekeeper/101.0.0+up3.9.0/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+annotations:
+  catalog.cattle.io/auto-install: rancher-gatekeeper-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: OPA Gatekeeper
+  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.25.0-0'
+  catalog.cattle.io/namespace: cattle-gatekeeper-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/provides-gvr: config.gatekeeper.sh.config/v1alpha1
+  catalog.cattle.io/rancher-version: '>= 2.6.0-0 < 2.7.0-0'
```

Comment:
catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'

Reply:
Fixed, PR was raised against 2.6 earlier, so this was missed.